pingcap/tidb
# server: fix region table http response

Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?

Issue Number: close #11918
Problem Summary: `/regions/{region}` API outputs many memory database tables.

### What is changed and how it works?

What's Changed: Skip memory dbs.


### Check List 

Tests 
- Manual test (add detailed scripts or steps below)
```
start 1pd+1tikv+1tidb
curl http://127.0.0.1:10080/regions/2
check output
```

### Release note 

- No release note
